ph_logger.c

```diff
@@ -689,6 +693,8 @@ static pid_t ph_logger_start_push_service(char *revision)
 		       "Initialized push service with pid %d by process with pid %d",
 		       getpid(), getppid());
 		pv_log(DEBUG, "Push service pushing logs for rev %s", revision);
+
+		current_process = "push_service_libthttp";
```

Comment:
this is a good idea. Can you change the other one in log.c to something like "ph_client_libthttp" so they are all using the same format?

Reply:
Sure, but are you sure that there are no others modules which are using that log?

Reply:
Yes, that is for all the requests that we are doing from the main fork. "ph_client_libthttp" its not very specific though and will fit well with almost anything.

Reply:
Done